lib/Basic/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
   set(UUID_LIBRARIES "rpcrt4.lib")
 else()
   find_package(UUID REQUIRED)
-  set(UUID_INCLUDE "-I${UUID_INCLUDE_DIRS}")
+  set(UUID_INCLUDE "${UUID_INCLUDE_DIRS}")
```

Comment:
This was wrong ... but would happen to work.  Why not directly use `${UUID_INCLUDE_DIRS}` below?